### PR TITLE
change to v7 data importer

### PR DESCRIPTION
**Pull request by ElaineRosenberg**
Modified all images to show data importer 7.0

Specified https://data-importer.neo4j.io/versions/0.7.0/?acceptTerms=true for the course so the images will always match what the user sees.

Modified section about getting latest.

Note: video does not need to change as it is the same behavior as v7.0 and it should not confuse the viewer

### CODE REVIEW

File: asciidoc/courses/importing-data/modules/2-using-data-importer/lessons/1-overview/lesson.adoc [hunk 1]
```diff
@@ -12,10 +12,11 @@ video::-QxpQ8FeD9M[youtube,width=560,height=315]
 [.transcript]
 == What is the Neo4j Data Importer?
 
-The Neo4j Data Importer can be found link:https://data-importer.graphapp.io/?acceptTerms=true[here^]:
+For this course, we use Neo4j Data Importer 7.0.
+The Neo4j Data Importer can be found link:https://data-importer.neo4j.io/versions/0.7.0/?acceptTerms=true[here^]:
```
Comment (emmaagh) [inline asciidoc/courses/importing-data/modules/2-using-data-importer/lessons/1-overview/lesson.adoc:16]: Users won't be able to easily connect to local databases on this link as it's neo4j.io, not graphapp.io - just checking that you've already discussed this and are happy with this limitation?

File: asciidoc/courses/importing-data/modules/2-using-data-importer/lessons/2-c-importing-CSV/lesson.adoc [hunk 2]
```diff
@@ -41,7 +41,7 @@ Uncompress this file. You should have these files:
 
 == Step 2: Open the Data Importer
 
-In a Web browser window, open the https://data-importer.graphapp.io/?acceptTerms=true[Neo4j Data Importer^]
+In a Web browser window, open the https://data-importer.neo4j.io/versions/0.7.0/?acceptTerms=true[Neo4j Data Importer^]
```
Comment (emmaagh) [inline asciidoc/courses/importing-data/modules/2-using-data-importer/lessons/2-c-importing-CSV/lesson.adoc:44]: Same here - users will have trouble connecting to local databases on the neo4j.io link - is this OK?

**Review by emmaagh — COMMENTED** [review]
Looks good, have just left a few comments checking some things.

Just to check we're on the same page, I think the best link for you to use would be https://data-importer.graphapp.io/versions/0.7.0?acceptTerms=true _except_ this link doesn't work (it redirects to latest 😞)

So that leaves the other 2 best links (with different tradeoffs) as:

- https://data-importer.graphapp.io/?acceptTerms=true - local DBs will work well, but app will become different from the course material over time
- https://data-importer.graphapp.io/versions/0.7.0?acceptTerms=true - app will remain consistent with the course material, but local DBs won't work well

File: asciidoc/courses/importing-data/includes/importer-notice.adoc [hunk 3]
```diff
@@ -1,7 +1,10 @@
...
+Neo4j Data Importer Early Access is evolving rapidly.
 
-https://data-importer.neo4j.io/versions/latest/
+There are two URLs for launching the **latest** version of Neo4j Data Importer:
 
-While the essence of the product remains the same, the UI will look different in a few places.
-When following along don’t be too surprised if the product looks different in a few places or ways of displaying information have changed or are different.
\ No newline at end of file
+* For connecting to a remote DBMS: https://data-importer.neo4j.io/?acceptTerms=true
+* For connecting to local or remote DBMSs: https://data-importer.graphapp.io/?acceptTerms=true
```
[the remark this answers is not in the document]
Comment (ElaineRosenberg) [reply asciidoc/courses/importing-data/includes/importer-notice.adoc]: Changed to http for local dbms

File: asciidoc/courses/importing-data/modules/2-using-data-importer/lessons/2-c-importing-CSV/lesson.adoc (hunk 2 above)
Comment (ElaineRosenberg) [reply asciidoc/courses/importing-data/modules/2-using-data-importer/lessons/2-c-importing-CSV/lesson.adoc:44]: For the course, they will be connecting to a sandbox, not local

File: asciidoc/courses/importing-data/includes/importer-notice.adoc [hunk 4]
```diff
@@ -1,7 +1,8 @@
-Neo4j Data Importer has launched in Beta and is evolving rapidly.
+Neo4j Data Importer has launched but new releases are published frequently.
+
 The latest version of Data Importer is here:
 
 https://data-importer.neo4j.io/versions/latest/
```
[the remark this answers is not in the document]
Comment (ElaineRosenberg) [reply asciidoc/courses/importing-data/includes/importer-notice.adoc]: fixed

File: asciidoc/courses/importing-data/modules/2-using-data-importer/lessons/1-overview/lesson.adoc (hunk 1 above)
Comment (ElaineRosenberg) [reply asciidoc/courses/importing-data/modules/2-using-data-importer/lessons/1-overview/lesson.adoc]: I changed it to 0.7.0 - will use  neo4j.io for the hands-on which uses a sandbox for the course.

Will also include info about using latest and graphapp.io.

File: asciidoc/courses/importing-data/modules/2-using-data-importer/lessons/1-overview/lesson.adoc [hunk 5]
```diff
@@ -12,10 +12,11 @@ video::-QxpQ8FeD9M[youtube,width=560,height=315]
 [.transcript]
 == What is the Neo4j Data Importer?
 
-The Neo4j Data Importer can be found link:https://data-importer.graphapp.io/?acceptTerms=true[here^]:
+For this course, we use Neo4j Data Importer 7.0.
```
[the remark this answers is not in the document]
Comment (ElaineRosenberg) [reply asciidoc/courses/importing-data/modules/2-using-data-importer/lessons/1-overview/lesson.adoc]: Changed it to 0.7.0

**Pull request merged**